reach4help/reach4help
# Add the world map to the landing page

## Type of Changes

|      | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: New feature     |
|  ✓    | :link: Update dependencies |

## Description
**This pull pull request**:
1- Adds new dependencies: `react-datamaps` and `d3`.
2- Adds  a`WorldMap` component to `path-to/landing-page/components`. That component displays the world map colored based on the number of users in each country. (The countries and number of users are hard coded for now)
3- Displays the component on the landing page.

## How it looks so far
![the world map](https://imgupload.io/images/2020/12/29/worldmap.jpg)

## Testing Guidelines
* From your terminal, pull down this branch with `git pull origin Feature/add-the-world-map-to-the-landing-page` and check that branch out with `git checkout Feature/add-the-world-map-to-the-landing-page` 
* Then `yarn` to install the newly added dependencies locally and `yarn start` to launch the app.
* Go to [http://localhost:3000/](http://localhost:3000/) to see the map on the landing page.

**Please let me know if anything needs to change. Thanks.**
